docs/roles/RoleNORManager.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: doc
 permalink: /roles/nor-manager
-title: OBO Role: New Ontology Request (NOR) Manager
```

Comment:
writing OBO in front of everything is redundant and a bit distracting, maybe better not to name roles OBO this and OBO that since it's implied that it's part of OBO

Reply:
alternative is to put quotes around all of this to make it valid yaml